enhancements/installer/aws-cross-account-dns-zone.md

```diff
@@ -218,6 +238,28 @@ An example IAM Trust Policy:
 
 ```
 
+#### External DNS Operator Details
+
+Prior to this enhancement, the [External DNS Operator API](https://github.com/openshift/external-dns-operator/blob/main/api)
+did not allow users to configure an AWS role ARN field. As a result, there wasn't a supported way to use the External
+DNS Operator to create DNS records in another AWS account within a shared VPC.
```

Comment:
Is there any built-in installer support ?  I thought ExternalDNS was a day-two installation option, meaning none of it is handled by the OpenShift installer.  How does this new functionality integrate with OpenShift Installer?

Reply:
You are right, ExternalDNS is a day-two operation, installed as a addon operator.

It doesn't integrate with the installer for this feature. It effectively provides the same feature as the installer and Ingress Operator implement together: support for creating a DNS record in an another zone.